client/package.json

```diff
@@ -103,6 +102,7 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.10",
+    "google-libphonenumber": "^3.2.31",
```

Comment:
this should be in the dependencies not on devDependencies because we need it in the production